site/blog/ecs-simplified.mdx

```diff
@@ -125,4 +90,21 @@ SELECT load_balancer_dns
 FROM ecs_simplified
 WHERE app_name = 'simple-express';
 ```
-Then you can check if the server is running on the `url:8088/health` address.
\ No newline at end of file
+Then you can check if the server is running on the `url:8088/health` address.
+
+## Low-level Access to Resources
+
+So the `aws_ecs_simplified` module simplifies things, right? But what if you still want the level of control you had when you were doing all the steps manually? You can't grow your app beyond limitations if you don't have access to all the small details. Let's say you want your ECS container to be able to use the AWS CLI to provision an EC2 instance, and for that purpose its IAM role needs `AmazonEC2FullAccess` policy to work properly. `aws_ecs_simplifiy` is good, but wasn't it better if we could just manage the lower-level details as well?
```

Comment:
Nit: want -> need. Hopefully people who *want* to just spend time fiddling with stuff are not using a tool to reduce the amount of stuff they fiddle with. ;)

Also restructuring a little bit that's a bit awkward-reading to me. Feel free to disregard that point, though.

```suggestion
So the `aws_ecs_simplified` module simplifies things, right? But what if you still need the level of control you had when you were doing all the steps manually? The traditional PaaS trade-off is that you can't grow your app beyond limitations as you don't have access to all the small details. The IaSQL approach is not limited in that way.

Let's say you want your ECS container to be able to use the AWS CLI to provision an EC2 instance, and for that purpose its IAM role needs `AmazonEC2FullAccess` policy to work properly. `aws_ecs_simplified` does not have a column to configure such a thing, but that doesn't mean we're stuck.
```